test/libyul/YulOptimizerTest.cpp

```diff
@@ -264,6 +265,24 @@ TestCase::TestResult YulOptimizerTest::run(ostream& _stream, string const& _line
 		LiteralRematerialiser::run(*m_context, *m_object->code);
 		UnusedFunctionParameterPruner::run(*m_context, *m_object->code);
 	}
+	else if (m_optimizerStep == "unusedFunctionReturnParameterPruner")
```

Comment:
Maybe use this step to test only the unused function return parameter pruner (this can be used to test that it does not modify small enough functions) and only use the "fullSuite" to test that it works efficiently?